cdi/pom.xml

```diff
@@ -88,7 +88,6 @@
         <dependency>
             <groupId>javax.enterprise</groupId>
             <artifactId>cdi-api</artifactId>
-            <scope>provided</scope>
         </dependency>
```

Comment:
Ok, so that's confirmation. Sorry about this, my mistake. I will repush it in a minute.